src/validator.ts

```diff
@@ -8,19 +8,9 @@ export class FormValidator {
         return emailRegex.test(email);
     }
 
-    validationNumber(): any {
-        const phoneInput = document.querySelector('input[name="Phone"]') as HTMLInputElement;
-        const phoneInputValue = phoneInput.value;
-        var selectElement = document.getElementById('phoneCountryCodeSelect') as HTMLSelectElement;
-        if (selectElement) {
-            var selectedOption = selectElement.options[selectElement.selectedIndex];
-            var selectedValue = selectedOption.value;
-            const phoneNumber = phoneInputValue.slice(selectedValue.length);
-            const phoneRegex = /^\d+$/;
-            return phoneRegex.test(phoneNumber);
-        } else {
-            console.error("Select element not found!");
-        }
+    validationNumber(phoneNumber: string): boolean {
+        const phoneRegex = /^\+\d{8,}$/;
+        return phoneRegex.test(phoneNumber);
```

Comment:
bardzo dobrze to wyglada ten refaktor. Wyrzucony ogrom kodu ktory nigdy nie powinien tam sie znalezc :)